lib/kamal/cli/app.rb

```diff
@@ -227,6 +227,24 @@ def logs
     end
   end
 
+  desc "open", "open app in a web browser"
+  def open
+    host = KAMAL.primary_role.proxy.hosts.first || KAMAL.primary_host
+    protocol = KAMAL.primary_role.ssl? ? "https" : "http"
+    cmd =
+      case RbConfig::CONFIG["host_os"]
+      when /mswin|mingw|cygwin/ then "start"
```

Comment:
Does Kamal work on Windows? I didn't find it mentioned in the  [Installation section](https://kamal-deploy.org/docs/installation/). If it does, maybe we could add. I haven't tried it yet.

Reply:
I just spun up a Windows VM, I don't know if the whole Kamal deploy flow works (I wasn't able to fully `bundle install` Rails), but `kamal apps open` by itself did work.